perl-Bootloader-testsuite/tests/test_interface/backup/fake_root1/boot/extlinux/extlinux.conf

```diff
@@ -0,0 +1,25 @@
+TIMEOUT 1000
+ONTIMEOUT localboot
+DEFAULT default
+MENU TITLE Boot menu
+PROMPT 0
+
+LABEL default
+         MENU LABEL Default
+         LINUX /boot/zImage
+         FDTDIR /boot/dtb/
+         FDT exynos5250-snow.dtb
```

Comment:
Good point. There should be either FDTDIR or FDT in a section. I have to sort this out.